contracts/dollar/contracts/Bonding.sol

```diff
@@ -15,281 +15,189 @@ import "./interfaces/IERC1155Ubiquity.sol";
 import "./utils/CollectableDust.sol";
 
 contract Bonding is CollectableDust {
-    using SafeERC20 for IERC20;
```

Comment:
Just don't modify any of the contracts please. I'm seeing 14 .sol files modified.